Makefile

```diff
@@ -9,7 +9,7 @@ TEST_FILES=$(shell find test/*.coffee)
 .pretestem: .all
 
 .app: $(APP_FILES)
-	node_modules/.bin/browserify $(APP_FILES) -o dist/twine.js -t coffeeify
```

Comment:
Was there no reason to use browserify before? Might want to use/ ditch `APP_FILES`